xcm/pallet-xcm/src/migration.rs

```diff
@@ -51,7 +49,7 @@ pub mod v1 {
 				VersionNotifyTargets::<T>::translate_values(translate);
 
 				log::info!("v1 applied successfully");
-				STORAGE_VERSION.put::<Pallet<T>>();
+				StorageVersion::new(1).put::<Pallet<T>>();
```

Comment:
I think we should pay attention in general to never use `STORAGE_VERSION.put`.  
There is one more case in `runtime/parachains/src/configuration/migration.rs`.

Reply:
> I think we should pay attention in general to never use STORAGE_VERSION.put.

Why?

Reply:
Because when you change `STORAGE_VERSION`, you will change the version the migration is writing to storage.